src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs

```diff
@@ -13,208 +13,208 @@
 
 namespace Microsoft.Identity.Web
 {
-    /// <summary>
-    /// Implementation for the downstream web API.
-    /// </summary>
-    public class DownstreamWebApi : IDownstreamWebApi
-    {
-        private readonly ITokenAcquisition _tokenAcquisition;
-        private readonly HttpClient _httpClient;
-        private readonly IOptionsMonitor<DownstreamWebApiOptions> _namedDownstreamWebApiOptions;
-        private readonly MicrosoftIdentityOptions _microsoftIdentityOptions;
-
-        /// <summary>
-        /// Constructor.
-        /// </summary>
-        /// <param name="tokenAcquisition">Token acquisition service.</param>
-        /// <param name="namedDownstreamWebApiOptions">Named options provider.</param>
-        /// <param name="httpClient">HTTP client.</param>
-        /// <param name="microsoftIdentityOptions">Configuration options.</param>
-        public DownstreamWebApi(
-            ITokenAcquisition tokenAcquisition,
-            IOptionsMonitor<DownstreamWebApiOptions> namedDownstreamWebApiOptions,
-            HttpClient httpClient,
-            IOptions<MicrosoftIdentityOptions> microsoftIdentityOptions)
-        {
-            _tokenAcquisition = tokenAcquisition;
-            _namedDownstreamWebApiOptions = namedDownstreamWebApiOptions;
-            _httpClient = httpClient;
+	/// <summary>
+	/// Implementation for the downstream web API.
+	/// </summary>
+	public class DownstreamWebApi : IDownstreamWebApi
+	{
+		private readonly ITokenAcquisition _tokenAcquisition;
+		private readonly HttpClient _httpClient;
+		private readonly IOptionsMonitor<DownstreamWebApiOptions> _namedDownstreamWebApiOptions;
+		private readonly MicrosoftIdentityOptions _microsoftIdentityOptions;
+
+		/// <summary>
+		/// Constructor.
+		/// </summary>
+		/// <param name="tokenAcquisition">Token acquisition service.</param>
+		/// <param name="namedDownstreamWebApiOptions">Named options provider.</param>
+		/// <param name="httpClient">HTTP client.</param>
+		/// <param name="microsoftIdentityOptions">Configuration options.</param>
+		public DownstreamWebApi(
+			ITokenAcquisition tokenAcquisition,
+			IOptionsMonitor<DownstreamWebApiOptions> namedDownstreamWebApiOptions,
+			HttpClient httpClient,
+			IOptions<MicrosoftIdentityOptions> microsoftIdentityOptions)
+		{
+			_tokenAcquisition = tokenAcquisition;
+			_namedDownstreamWebApiOptions = namedDownstreamWebApiOptions;
+			_httpClient = httpClient;
 #pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
This warning suppression might be able to be removed now that strict null types is enabled.